src/components/LibraryCardForm/LibraryCardForm.tsx

```diff
@@ -11,12 +10,36 @@ import FormField from '../FormField/FormField';
 import ApiErrors from '../ApiErrors/ApiErrors';
 import config from '../../../appConfig';
 
-class LibraryCardForm extends React.Component {
+interface LibraryCardFormState {
+  agencyType: string;
+  csrfToken: string;
+  focusOnResult: boolean;
+  formProcessing: boolean;
+  formEntrySuccessful: boolean,
+  apiResults: any;
+  fieldErrors: any;
+  patronFields: any;
+}
+
+class LibraryCardForm extends React.Component<{}, LibraryCardFormState> {
+  dynamicSection = React.createRef<HTMLDivElement>();
+  stateName = React.createRef<any>();
```

Comment:
Why `any` for all of these; wouldn't they all be input elements?

Reply:
True, this I did forget to do.

Reply:
I'm having trouble with my linter so I updated this in the PR with the updated linter:
https://github.com/NYPL/nypl-library-card-app/pull/45/files?file-filters%5B%5D=.json&file-filters%5B%5D=.scss&file-filters%5B%5D=.tsx#diff-18c67e654e4a49eef2d852390f9dfccbR27